src/tests/ftest/recovery/pool_membership.py

```diff
@@ -239,26 +239,47 @@ def test_dangling_pool_map(self):
         dmg_command = self.get_dmg_command()
         dmg_command.system_stop()
 
-        # 3. Manually remove /mnt/daos0/<pool_uuid>/vos-0 from rank 0 node.
+        # 3. Manually remove /<scm_mount>/<pool_uuid>/vos-0 from rank 0 node.
         rank_0_host = NodeSet(self.server_managers[0].get_host(0))
-        rm_cmd = f"sudo rm /mnt/daos0/{pool.uuid.lower()}/vos-0"
+        scm_mount = self.server_managers[0].get_config_value("scm_mount")
+        rm_cmd = f"sudo rm {scm_mount}/{pool.uuid.lower()}/vos-0"
         if not run_remote(log=self.log, hosts=rank_0_host, command=rm_cmd).passed:
             self.fail(f"Following command failed on {rank_0_host}! {rm_cmd}")
 
         # 4. Enable and start the checker.
         self.log_step("Enable and start the checker.")
         dmg_command.check_enable(stop=False)
-        dmg_command.check_start()
 
-        # 5. Query the checker and verify that the issue was fixed.
         errors = []
         query_msg = ""
-        for _ in range(20):
-            check_query_out = dmg_command.check_query()
-            if check_query_out["response"]["status"] == "COMPLETED":
-                query_msg = check_query_out["response"]["reports"][0]["msg"]
+
+        # If we start the checker right after enabling it, the checker may not detect any
+        # fault (can't reproduce manually). If it happens, stop and restart the checker.
```

Comment:
Is that a bug?

Reply:
I'm not sure. It occurs intermittently during automated script. I can't reproduce it manually.

Reply:
It might be worth asking a relevant developer for insight, in case a similar issue occurs in production. And I think it's better to now workaround an issue by just restarting, since that could lead to test gaps.

Reply:
I talked to the developers about this and they are aware of this issue and working on it. I removed the restart and added sleep because they recommended this way.